packages/forge/blocks/perplexity/src/index.ts

```diff
@@ -0,0 +1,15 @@
+import { createBlock } from "@typebot.io/forge";
+import { createChatCompletion } from "./actions/createChatCompletion";
+import { auth } from "./auth";
+import { PerplexityDarkLogo, PerplexityLightLogo } from "./logo";
+
+export const perplexityBlock = createBlock({
+  id: "perplexity",
+  name: "Perplexity",
+  tags: ["ai", "chat", "completion", "sonar", "sonar-reasoning"],
+  LightLogo: PerplexityLightLogo,
+  DarkLogo: PerplexityDarkLogo,
+  auth,
+  actions: [createChatCompletion],
+  docsUrl: "https://docs.typebot.io/editor/blocks/integrations/perplexity",
```

Comment:
You need to create the doc then if you provide it! Otherwise, you can leave it empty for now